index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="CSS/style.css"></link>
+
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <div class="row">
+
+          <div class="column">
```

Comment:
Looks like there may be some unclosed divs here.

Delete the code for the second column and by placing your cursor on the opening tag of this first `column` element, your editor will underline the closing tag.

Then use the knowledge of where the opening and closing tags are to duplicate the full element and then work from there.